src/locales/en.json

```diff
@@ -5,7 +5,24 @@
       "cancel": "Cancel",
       "delete": "Delete",
       "save": "Save",
-      "signIn": "Sign In"
+      "signIn": "Sign In",
+      "signUp": "Sign Up",
+      "register": "Register",
+      "home": "Home"
+    },
+
+    "text": {
```

Comment:
in en.json. Pls using key same like page. Ex: login: {signIn: "SignIn"}, signUp: {}